pre_commit/languages/node.py

```diff
@@ -98,8 +99,9 @@ def install_environment(
         )
         lang_base.setup_cmd(prefix, local_install_cmd)
 
-        _, pkg, _ = cmd_output('npm', 'pack', cwd=prefix.prefix_dir)
-        pkg = prefix.path(pkg.strip())
+        _, pkg, _ = cmd_output('npm', 'pack', '--json', cwd=prefix.prefix_dir)
+        pkg_json = json.loads(pkg)
+        pkg = prefix.path(pkg_json[0].get('filename'))
```

Comment:
`.get` will "silently" fail when the key is missing -- use brackets to access keys in a dictionary